commercetools-java-client-internal-test/src/main/java/io/sphere/sdk/client/HttpClientAdapterTest.java

```diff
@@ -105,6 +105,6 @@ private void checkBodyRequest(final int port, final HttpRequestBody requestBody,
         client.close();
         final String receivedHttpHeadersAndBody = future.join();
         LoggerFactory.getLogger(HttpClientAdapterTest.class).info(receivedHttpHeadersAndBody);
-        assertThat(receivedHttpHeadersAndBody).contains("Content-Length: " + length);
+        assertThat(receivedHttpHeadersAndBody).containsIgnoringCase("Content-Length: " + length);
```

Comment:
That's interesting, I didn't expect that HTTP headers are case insensitive...

Reply:
Yep according to `RFC 2616` they are case insensitive and when i updated the client the test failed because they became lower case, so i had to ignore the case.